### DISCUSSION TITLE
OOF point in time predictions

### DISCUSSION THREAD

**Discussion by BigW** [amazon-science/chronos-forecasting#147] [Q&A] [answered]
Hi
For context, I am trying to extract point in time predictions for financial time series using AG Chronos.
For each time/date I would ideally like to output and store a prediction for the defined step ahead.
Using S&P daily data for example I would define the daily closing price as the target value and try to predict that one day ahead of time. 
The ability to generate and extract the predictions with dates is an important part of testing to evaluate how the model performs on a stand alone basis or in combination with other models.

Prior feedback from @shchur confirmed that the ensemble AG model does not output the OOF predictions.

My questions are:
1. Does AG Chronos output OOF predictions in any way that makes it possible for me to store those in a df/csv?
2. Are there any similar examples already reported and outlined - I have not seen any yet?
Thank you

**Comment by shchur** [comment, marked as answer]
Hi @BigW, all predictions generated by the the Chronos are OOF by their nature since the model was pre-trained on other datasets.

The Chronos implementation in this repo does not natively support dataframes but the implementation in AutoGluon does. We can continue the discussion in this thread https://github.com/autogluon/autogluon/issues/3525.
